src/RA_Achievement.h

```diff
@@ -122,7 +115,7 @@ class Achievement
 	/*const*/ AchievementSetType m_nSetType;
 
 	AchievementID m_nAchievementID;
-	std::vector<ConditionSet> m_vConditions;
+	ConditionSet m_vConditions;
```

Comment:
`ConditionSet` was renamed `ConditionGroup` and a group of `ConditionGroup`s is now a `ConditionSet`. `ConditionSet` contains the logic for supporting OR'd groups.